Random.py

```diff
@@ -46,21 +46,4 @@ def print_students(self):
 manager = StudentManager()
 manager.add_student("Alice", 20, 1)
```

Comment:
The indentation of the lines in the `update_player_pos` function is inconsistent. This can lead to runtime errors. ```diff -    x, y = player_pos -    if direction == "up": -        y -= 1 -    elif direction == "down": -        y += 1 -    elif direction == "left": -        x -= 1 -    elif direction == "right": -        x += 1 +    x, y = player_pos +    if direction == "up": +        y -= 1 +    elif direction == "down": +        y += 1 +    elif direction == "left": +        x -= 1 +    elif direction == "right": +        x += 1 ``` ---